src/config.ts

```diff
@@ -0,0 +1,84 @@
+import {cosmiconfigSync} from 'cosmiconfig';
+import {Validator} from 'jsonschema';
+import {merge} from 'lodash';
+
+export interface SnowpackConfig {
+  options: {
+    babel?: boolean;
+    clean?: boolean;
+    dest?: string;
+    exclude?: string[];
+    externalPackage?: string[];
+    include?: string[];
+    nomodule?: string;
+    nomoduleOutput?: string;
+    noSourceMap?: boolean;
+    optimize?: boolean;
+    remotePackage?: string[];
+    remoteUrl?: string;
+    strict?: boolean;
+  };
```

Comment:
Wanted to point out that all the CLI options are under an `options` namespace. That’s because the way cosmiconfig works, it allows users to add configuration inside `package.json` under the `snowpack` namespace. You may already realize that’s where `webDependencies` lives!

For that reason, I’d recommend namespacing all the CLI options for a more future-proof thing. I’m neutral on `options`, but I think we should namespace these _somehow_.

Reply:
Right—it’s the same principle as namespacing in, say, an API or GraphQL. Once something’s at the top-level, it’s tough to change later.  That’d be adding `13` options to the top-level namespace we’d have to support for a while.

🤔 What about `buildOptions`? That sounds a little less dumb. And it happens at build time.

Reply:
Yea, I think that's the most technically correct description, but most of our users consider this an install tool vs. a build tool. So maybe "installOptions"? As in, here's how you configure how these packages are installed.

But then of course any option is an installOption, when the tool is a package installer. But TypeScript is a compiler and they get away with `compilerOptions`, so maybe that's fine!

I'm not picky here, but I do worry a bit about showing an early user the word "build" when they start to configure Snowpack.